source-mysql/replication.go

```diff
@@ -199,7 +199,10 @@ func (rs *mysqlReplicationStream) run(ctx context.Context) error {
 			// that the server's `binlog_format` is not set correctly. Even when it's correctly set
 			// to `ROW`, Query events will still be sent for DDL statements, so we don't want to
 			// return an error here, but we do want these logs to be fairly visible.
-			logrus.WithField("data", data).Info("Query Event")
+			logrus.WithFields(logrus.Fields{
+				"event": data,
```

Comment:
super minor quibble: when the logging format is JSON (always, unless running the connector manually), I'd think that the query would then be duplicated in the logs. Maybe just log `data`?

Reply:
I hadn't actually thought about how it looks as JSON, I was just making it look nicer in the human-readable output. However I think this might still be a good change, because the `Query` field of the `QueryEvent` struct is a `[]byte`, and so without this edit the JSON-formatted output looks like:

```json
{
  "data": {
    "SlaveProxyID": 275,
    "ExecutionTime": 0,
    "ErrorCode": 0,
    "StatusVars": "AAAAAAABIACgRQAAAAAGA3N0ZAQhACEA/wAS/wA=",
    "Schema": "dGVzdA==",
    "Query": "QkVHSU4=",
    "GSet": null
  },
  "level": "info",
  "msg": "Query Event",
  "time": "2022-03-04T18:19:39-06:00"
}
```

I think it's helpful to not make the user copy-paste queries into `base64 -d` in order to tell whether a given QueryEvent should be cause for concern.

Reply:
> I think it's helpful to not make the user copy-paste queries into base64 -d in order to tell whether a given QueryEvent should be cause for concern.

:+1: